docs/achtergronddocumentatie/artefacten/0534.md

```diff
@@ -1,95 +1,43 @@
+- Als de identiteit van de klant bekend is (en er dus authenticatie heeft plaatsgevonden) wordt er wel een klantcontact aangemaakt.
+- 
+## 2. Klantcontact wordt geregistreerd met behulp van [F2173 - registreer klantcontact](./2173)
 
-** Aanmaken **
-| Objecttype - Attribuut - Relatie| Voorbeeld waarde | Toelichting |
-| :----------- | :----------- | :----------- |
-| `Klantcontact` | | |
-| - `ID` | 5452f450-e940-4996-a353-37f4a7ec8710 | Automatisch ingevuld door consumer |
-| - `Nummer` | 23009182 | Automatisch ingevuld door provider |
-| - `Kanaal` | Telefoon | Automatisch ingevuld door consumer | 
-| - `Onderwerp` | Voorwaarden vergunningsvrije dakkapel | Ingevuld door gebruiker |
-| - `Inhoud` | Klant wil weten onder welke voorwaarden er geen vergunning nodig is voor een dakkapel. | Ingevuld door gebruiker |
-| - `Initiator` | Klant | Automatisch ingevuld door consumer |
-| - `Indicatie contact gelukt` | Ja | Automatisch gevuld door consumer |
-| - `Taal` | 'Nederlands` | Prefill door consumer wijzigbaar door gebruiker |
-| - `Vertrouwelijkheid` | Nee | Prefill door consumer wijzigbaar door gebruiker |
-| - `Plaatsgevonden op` | 08-03-2023 10:00 | Prefill door consumer wijzigbaar door gebruiker |
-| - `Geregistreerd op` | 08-03-2023 10:02 | Automatisch ingevuld door provider |
-| - `Betrokkenen` | | |
-| - - `Betrokkene bij klantcontact` | | | 
-| - - - `ID` | cba5de29-acc1-4ca3-a24e-3150c83d31eb | Automatisch ingevuld door consumer |
-| - - - `Rol` | Klant | Automatisch ingevuld door consumer |
-| - - - `Contactnaam` | Naam klant | Ingevuld door gebruiker | 
-| - - - `Geregistreerd op` | 08-03-10:02 | Automatisch ingevuld door provider |
-| - `Actoren` | | | 
-| - - `Actor bij klantcontact` | | |
-| - - - `Actorid` | a255acc8-b7b9-4db3-94aa-ecba5aec9f5f | Automatisch ingevuld door consumer |
+- In het klantcontact wordt de vraag beschreven in de *inhoud*
+- Eventueel wordt er een onderwerp ingevuld in de *onderwerp* van het klantcontact
+- Het vastleggen van klant-gegevens is beschreven in de paragraaf [Vastleggen van de betrokkeneBijKlantcontact](./2173.md#vastleggen-betrokkenebijklantcontact)
 
+## 3. Overname van gesprek door de afdeling vergunningen
 
-<!-- ## 2. Medewerker1 maakt voor de afdeling vergunningen een interne taak aan
+- De KCC-medewerker belt de collega vergunningen-medewerker om de klant door te verbinden
+- Telefonisch wordt de verwachte handeling en het *nummer* van het `klantcontact` doorgegeven aan de vergunningen-medewerker
+- De vergunningen-medewerker opent een scherm om het opgegeven klantcontact te selecteren ([F3896 - Zoek klantcontact](./3896.md)) 
+- Na selectie wordt de informatie over het `klantcontact` getoond en de naamgegevens van de klant uit `betrokkene bij klantcontact` [F4936 - Zoek betrokkeneBijKlantcontact](./4936) of eventueel uit de daaran gerelateerde `partij` [F0138 - Vraag partij op](./0138) opgehaald en getoond.
```

Comment:
De voorwaarde voor vastlegging van een klantcontact dat een gekende klant betrokken is (regel 17) betekent volgens mij dat altijd partijgegevens moeten worden opgehaald...

Reply:
In dat geval betekent dat dat ik alleen zaken met de gemeente kan doen als mijn gegevens voor hergebruik worden vastgelegd. Dat moeten we dan heel duidelijk maken.
Ik was in de veronderstelling dat ik me wel kan authenticeren en dus bekend is wie ik ben, maar dat ik mijn gegevens niet voor hergebruik wil laten opslaan als partij. Gaarne bespreken.

Reply:
Dat je alleen zaken kan doen met de gemeente na vastlegging voor hergebruik zou ik niet willen betogen, net zomin als het registreren van een klantcontact noodzakelijk zou zijn om iets van de gemeente gedaan te krijgen ;). Ik zie de volgende mogelijkheden als het gaat om eventuele registratie van een klantcontact en gerelateerde gegevens naar aanleiding van contact met de gemeente:

1. er heeft geen authenticatie plaatsgevonden en er is geen opvolging nodig. Omdat we directe overdracht aan een collega beschouwen binnen de context van één klantcontact, beschouwen we doorverbinden niet als opvolging. Registratie van een klantcontact in deze situatie dient geen (inzage-, hergebruiks- of opvolgings)doel. Er wordt in dit geval dus geen klantcontact geregistreerd.
2. er heeft geen authenticatie plaatsgevonden maar er is opvolging nodig. Nu wordt een klantcontact geregistreerd met in betrokkene de contactgegevens van de partij/vertegenwoordiger. Er wordt ook een interne taak voor de opvolging geregistreerd. Partij speelt geen rol.
3. er heeft authenticatie plaatsgevonden, er is wel of geen opvolging nodig en de klant/vertegenwoordiger geeft géén toestemming voor hergebruik van gegevens en heeft die eerder ook niet gegeven: er ontstaat een klantcontact met bij betrokkene eventueel contactgegevens voor opvolging van het klantcontact. Deze betrokkene wordt gekoppeld aan een, behoudens een naar een extern persoonsregister verwijzende partij-identificator en een partijnummer, _lege_ partij.
4. er heeft authenticatie plaatsgevonden, er is wel of geen opvolging nodig en de klant/vertegenwoordiger geeft toestemming voor hergebruik van gegevens of heeft die eerder gegeven: er ontstaat een klantcontact met bij betrokkene eventueel specifiek voor opvolging van het klantcontact verstrekte contactgegevens. Deze betrokkene wordt gekoppeld aan een nieuw aangemaakte, bestaande of aangevulde partij waarin 'default' contactgegevens zijn vastgelegd.

Binnen de beperkingen van deze casus passen niet alle hierboven beschreven mogelijkheden:
- Mogelijkheid 1 past binnen de casus, maar dan wordt géén klantcontact geregistreerd (zie regel 15).
- Omdat deze casus de situatie beschrijft dat de gestelde vraag binnen de context van één klantcontact beantwoord kan worden, past mogelijkheid 2 niet binnen de casus.
- Mogelijkheden 3 en 4 passen allebei binnen de casus, maar in beide gevallen is sprake van koppeling aan een nieuwe of bestaande partij.

Ergo (volgens mij): áls binnen de context van deze casus een klantcontact wordt geregistreerd, is daarbij altijd een partij betrokken.

Reply:
Ik heb het vraagstuk toegevoegd, een vraagstuk rondom het klantcontact aangevuld en met de bovenstaande informatie  de betrokken casus en 2 functies aangepast.